pyomo/contrib/mindtpy/tests/test_mindtpy.py

```diff
@@ -161,11 +164,129 @@ def test_OA_Proposal_with_int_cuts(self):
                       mip_solver=required_solvers[1],
                       nlp_solver=required_solvers[0],
                       add_integer_cuts=True,
-                      integer_to_binary=True)
+                      integer_to_binary=True  # if we use lazy callback, we cannot set integer_to_binary True
+                      )
+
+            # self.assertIs(results.solver.termination_condition,
+            #               TerminationCondition.optimal)
+            self.assertAlmostEqual(value(model.obj.expr), 0.66555, places=2)
+
+    def test_OA_OnlineDocExample(self):
+        with SolverFactory('mindtpy') as opt:
+            model = OnlineDocExample()
+            print('\n Solving problem with Outer Approximation')
+            opt.solve(model, strategy='OA',
+                      mip_solver=required_solvers[1],
+                      nlp_solver=required_solvers[0]
+                      )
+            self.assertAlmostEqual(value(model.objective.expr), 3, places=2)
+
+    # the following tests are used to improve code coverage
+    def test_OA_OnlineDocExample2(self):
```

Comment:
Would prefer if these titles indicated what was being tested. E.g. `iteration limit`

Reply:
I agree with @qtothec: it s best if the test names / documentation give an indication of what is being tested.